meta-iotqa/scripts/contrib/licensetree-whitelist.txt

```diff
@@ -0,0 +1,35 @@
+virtual/update-alternatives
+make
+libtool-cross
+ptest-runner-dev
+ptest-runner
+virtual/gettext
+perl
+base-files
+cryptodev-linux
+shadow
+shadow-sysroot
+update-rc.d
+initscripts
+bash-completion
+iptables
+nftables
+ppp
+wpa-supplicant
+cpp
+ofono
+bluez5
+connman
+util-linux
+udev
+libgcc-initial
+linux-libc-headers
+python-core
+python3-core
+kernel-4.9.20-yocto-standard
```

Comment:
Using the linking information would help a lot, yes. However, it doesn't solve the (possible) issues with non-C components, such as node.js or Python libraries. Maybe those recipes could then be blacklisted based on the inherited classes or something?